client/utils/auth.py

```diff
@@ -318,13 +327,13 @@ def get_info(assignment, access_token):
     response.raise_for_status()
     return response.json()['data']
 
-def display_student_email(assignment, access_token):
+def display_student_email(assignment, access_token, warn=True):
     try:
         email = get_info(assignment, access_token)['email']
         print('Successfully logged in as', email)
         return email
-    except Exception:
-        log.warning('Could not get student email', exc_info=True)
+    except:
```

Comment:
Why the change from `Exception` to catching anything? That means e.g. keyboard interrupts will be swallowed